R/calculateDistance.R

```diff
@@ -46,7 +50,8 @@ setGeneric("calculateDistance", signature = c("x"),
 #' @export
 setMethod("calculateDistance", signature = c(x = "ANY"),
     function(x, FUN = stats::dist, ...){
-        do.call(FUN, c(list(x),list(...)))
+        .Deprecated("directly 'FUN' argument's specified function")
```

Comment:
What is the output of this? The message in itself seems to be a bit hard to understand. Can you make the message more verbose?

Reply:
It is 

```
> calculateDistance(se)
         Sample1  Sample2  Sample3  Sample4  Sample5
Sample2 4.242641                                    
Sample3 5.385165 1.732051                           
Sample4 5.385165 4.358899 4.000000                  
Sample5 3.464102 3.162278 3.605551 2.236068         
Sample6 4.123106 1.000000 2.000000 4.242641 3.000000
Warning message:
'calculateDistance' is deprecated.
Use 'directly 'FUN' argument's specified function' instead.
See help("Deprecated") 
```

The problem is that `FUN` contains the function, not function name, so I'm not sure what is good way

Reply:
It is also possible to create own message. I think its better so we get rid off those extra quotation marks
```

> calculateDistance(se)
         Sample1  Sample2  Sample3  Sample4  Sample5
Sample2 4.242641                                    
Sample3 5.385165 1.732051                           
Sample4 5.385165 4.358899 4.000000                  
Sample5 3.464102 3.162278 3.605551 2.236068         
Sample6 4.123106 1.000000 2.000000 4.242641 3.000000
Warning message:
This is also possible... 
```

Reply:
Current warning message

```
> calculateDistance(se)
         Sample1  Sample2  Sample3  Sample4  Sample5
Sample2 4.242641                                    
Sample3 5.385165 1.732051                           
Sample4 5.385165 4.358899 4.000000                  
Sample5 3.464102 3.162278 3.605551 2.236068         
Sample6 4.123106 1.000000 2.000000 4.242641 3.000000
Warning message:
'calculateDistance' is deprecated. 
Instead, use directly the function that is specified by 'FUN' argument. 
See help('Deprecated') 
```